tests/python/04_swizzle/main.py

```diff
@@ -45,108 +44,50 @@ def runKernel(opt):
         if remainder == 3:
             reference.append(idx-2)
 
-    if xclSyncBO(opt.handle, boHandle, xclBOSyncDirection.XCL_BO_SYNC_BO_TO_DEVICE, DATA_SIZE, 0):
-        return 1
 
-    p = xclBOProperties()
-    bodevAddr = p.paddr if not (xclGetBOProperties(opt.handle, boHandle, p)) else -1
+    obj.sync(pyxrt.xclBOSyncDirection.XCL_BO_SYNC_BO_TO_DEVICE, size, 0)
```

Comment:
It would be nicer to have a specific namespace for`xcl`, don't you  think?
```suggestion
    obj.sync(pyxrt.xcl.bo_sync_direction.to_device, size, 0)
```

Reply:
@stsoe we should consider this for C++ bindings?